PULL_REQUEST_TEMPLATE

```diff
@@ -16,7 +16,8 @@ Don't forget to add a prefix to the pr title in this format
 ### Checklist
 
 - [ ] I performed some relevant testing on a real device and/or simulator
-- [ ] I wrote Unit Tests that confirm the expected behaviour
+- [ ] I wrote Unit Tests that confirm the expected behavior
 - [ ] I updated only the [english localization source files](Client/Ecosia/L10N/es.lproj) if needed
 - [ ] I added the `// Ecosia:` helper comments where needed
-- [ ] I included documentation updates to the coding standards or Confluence doc, when needed
\ No newline at end of file
+- [ ] I included documentation updates to the coding standards or Confluence doc when needed
+- [ ] I updated `Core` back to the `main` reference with a new revision if needed
```

Comment:
Non-blocking suggestion:

Maybe we could make it more generic like
```suggestion
- [ ] I am using the correct version from [core](https://github.com/ecosia/ios-core), including any necessary changes for this ticket and pointing to `main` instead of a feature branch
```